src/sources/gcp_pubsub.rs

```diff
@@ -294,8 +304,35 @@ impl SourceConfig for PubsubConfig {
         Ok(Box::pin(source))
     }
 
-    fn outputs(&self, _global_log_namespace: LogNamespace) -> Vec<Output> {
-        vec![Output::default(DataType::Log)]
+    fn outputs(&self, global_log_namespace: LogNamespace) -> Vec<Output> {
+        let log_namespace = global_log_namespace.merge(self.log_namespace);
+        let schema_definition = self
+            .decoding
+            .schema_definition(log_namespace)
+            .with_standard_vector_source_metadata()
+            .with_source_metadata(
+                PubsubConfig::NAME,
+                Some(LegacyKey::Overwrite(owned_value_path!("timestamp"))),
+                &owned_value_path!("timestamp"),
+                Kind::timestamp().or_undefined(),
+                Some("timestamp"),
+            )
+            .with_source_metadata(
+                PubsubConfig::NAME,
+                Some(LegacyKey::InsertIfEmpty(owned_value_path!("attributes"))),
+                &owned_value_path!("attributes"),
+                Kind::object(Collection::empty()),
```

Comment:
```suggestion
                Kind::object(Collection::empty().with_unknown(Kind::bytes())),
```
It looks like any of the values here would be Bytes?

```rust
        let attributes = Value::Object(
            message
                .attributes
                .into_iter()
                .map(|(key, value)| (key, Value::Bytes(value.into())))
                .collect(),
        );
```